problems/[230716] Flood Fill.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/flood-fill
+ */
+
+/**
+ * @param {number[][]} image
+ * @param {number} sr
+ * @param {number} sc
+ * @param {number} color
+ * @return {number[][]}
+ */
+const floodFill = (image, sr, sc, newColor, color) => {
+  const firstColor = color === undefined ? image[sr][sc] : color;
```

Comment:
`||` 연산자를 사용해 더 간단하게 표현도 가능할 것 같습니다.
```suggestion
  const firstColor = (color || image[sr][sc]);
```